cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data-handler.handler.ts

```diff
@@ -116,31 +187,29 @@ export async function handler(_: APIGatewayProxyEvent): Promise<APIGatewayProxyR
 
   const s3Params = {
     Bucket: 'opendataplatformapistaticdata',
-    Key: 'pwsid_lead_connections.geojson',
+    Key: 'pwsid_lead_connections_even_smaller.geojson',
```

Comment:
potentially comment on what _even_smaller is here.

Reply:
Done